addons/html_builder/static/src/website_builder/plugins/collapse_plugin.js

```diff
@@ -0,0 +1,89 @@
+import { Plugin } from "@html_editor/plugin";
+import { registry } from "@web/core/registry";
+
+export class CollapsePlugin extends Plugin {
+    static id = "collapse";
+
+    resources = {
+        on_snippet_dropped_handlers: this.onSnippetDropped.bind(this),
+        on_cloned_handlers: this.onCloned.bind(this),
+        dropzone_selector: [
+            {
+                selector: ".accordion-item",
+                dropLockWithin: ".accordion",
```

Comment:
Why? Is it a request from someone? It seems that in master, you can can drag an accordion item in another accordion